dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -201,70 +201,45 @@
                   </div>
                 </div>
 
-                <div
-                  v-for="financialServiceOption of confirmedSelectedFinancialServiceOptions"
-                  :key="financialServiceOption"
-                  :data-test="financialServiceOption.value"
-                  class="uploadFormSection"
-                >
-                  <div class="flex w-full">
-                    <div class="p-3 topicLabel">
-                      <h3 :id="financialServiceOption.value" class="anchor title">
-                        {{ financialServiceOption.label }}
-                      </h3>
-                    </div>
-
-                    <PrimeButton
-                      @click="removeKpisSection(financialServiceOption.value)"
-                      label="REMOVE THIS SECTION"
-                      data-test="removeSectionButton"
-                      class="p-button-text ml-auto"
-                      icon="pi pi-trash"
-                    ></PrimeButton>
-                  </div>
-
-                  <FormKit
-                    v-if="financialServiceOption.value !== 'assetManagementKpis'"
-                    :name="financialServiceOption.value"
-                    type="group"
+                <FormKit v-model="kpiSections" name="kpiSectionsModel" type="group">
```

Comment:
The name kpiSectionsModel does not seem fit to me. I don't get the Model suffix.